core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -221,44 +222,44 @@ private static boolean validateLocal(String target, String name, String[] args)
     int validationCount = 0;
     int failureCount = 0;
     Collection<ValidationTask> tasks = TARGET_TASKS.get(target);
-    for (ValidationTask task : tasks) {
+    List<ValidationTask> tasksToRun = new ArrayList<>();
+    Options options = new Options();
+    for (ValidationTask task: tasks) {
       String taskName = TASKS.get(task);
       if (name != null && !taskName.startsWith(name)) {
         continue;
       }
-
-      CommandLine cmd;
-      try {
-        cmd = task.parseArgsAndOptions(args);
-      } catch (InvalidArgumentException e) {
-        System.err.format("Invalid argument for task %s.%n", taskName);
-        continue;
-      }
-      Map<String, String> optionsMap = new HashMap<>();
-      for (Option opt : cmd.getOptions()) {
-        optionsMap.put(opt.getOpt(), opt.getValue());
-      }
-      System.out.format("Validating %s...", taskName);
+      task.addOptions(options);
```

Comment:
It would be better to do addOptions in the registerTask function. With that you will get options for all tasks instead of for a particular target, and then we can use `HelpFormatter.printHelp(USAGE, OPTIONS)` to display help message for all the available options

Reply:
Done.